### DISCUSSION TITLE
投票选出你更关心的游戏开发技术的主题

### DISCUSSION THREAD

**Discussion by gonglei007** [gonglei007/GameDevMind#2] [Polls]
为你感兴趣的主题投上一票，让我们更多的来讨论它。

**Comment by ZQlby** [comment]
更关注应该是多选呀，单选就是最关注了呀🐶。btw，很有用，很感谢🌹～

**Reply by gonglei007** [reply]
感谢参与，也感谢建议。头一次使用Github的投票，缺少经验。以后改进。

**Comment by WangShuXian6** [comment]
服务端吧，这个在国内外几乎是0.都是业内自己玩

**Comment by wanzaixiaoxin** [comment]
产品和商业化感觉应该分开，核心先有一个好的产品，商业化偏后期一些

**Reply by gonglei007** [reply]
嗯，咱们这里的“商业化”仅限于商业化过程中所涉及的技术研发这边相关的事项。
